src/Common/CSharp/Analysis/CallExtensionMethodAsInstanceMethodAnalysis.cs

```diff
@@ -110,14 +110,14 @@ MemberAccessExpressionSyntax CreateNewMemberAccessExpression()
                 case SyntaxKind.GenericName:
                     {
                         return SimpleMemberAccessExpression(
-                            ParenthesizedExpression(argument.Expression),
```

Comment:
I think that parentheses could be necessary sometimes. Better would be to keep parentheses and add `SimplifiedAnnotation` which will take care of removing parentheses if redundant. There is extension method `Parenthesize` which will parenthesize the expression and add `SimplifierAnnotation` by default.

Reply:
Thank you, I put there the extension method Parenthesize.